Jenkinsfile

```
@@ -3,10 +3,10 @@
 buildPlugin(useContainerAgent: true,
         configurations: [
                 [platform: 'linux', jdk: '11'],
-                [platform: 'windows', jdk: '8'],
+                [platform: 'windows', jdk: '11'],
 
                 // testing the Guava & Guice bumps
                 // https://github.com/jenkinsci/jenkins/pull/5707
                 // https://github.com/jenkinsci/jenkins/pull/5858
-                [ platform: "linux", jdk: "8", jenkins: '2.321' ]
+                [ platform: "linux", jdk: "17", jenkins: '2.361.4' ]
```

Usually you want to test a different Jenkins version than the default so that you know if other problems are detected by the newer Jenkins version
```suggestion
                [ platform: "linux", jdk: "17", jenkins: '2.387.2' ]
```

---

may I know why you've changed the Jenkins version to 2.387.2?
I just updated the Jenkins version to the recommended one in the documentation.

@MarkEWaite But yet the check is failed

---

> may I know why you've changed the Jenkins version to 2.387.2?

The Jenkins version 2.361.4 is the value that is used already in the line that tests Jenkins on Linux.  The `jenkins.version` value is the default when no `jenkins:` parameter is defined.

I used a newer version (2.400) in the most recent build so that it would test a wider range of functionality with the same build process.